django_rq/utils.py

```diff
@@ -111,7 +111,7 @@ def get_scheduler_statistics():
             connection = first_sentinel.connection_pool.connection_kwargs
         else:
             connection = queue.connection.connection_pool.connection_kwargs
-        conn_key = f"{connection['host']}:{connection.get('port', 6379)}/{connection.get('db', 0)}"
+        conn_key = f"{connection.get('host', 'NOHOST')}:{connection.get('port', 6379)}/{connection.get('db', 0)}"
```

Comment:
Hey, I'm not sure why falling back to `NOHOST` would fix this issue. Care to elaborate on why the Redis connection has no `host` property?